data/authors.yaml

```diff
@@ -7,3 +7,4 @@ authors:
         - https://www.linkedin.com/in/marekzaluski
     - id: lisa-tagliaferri
       name: Lisa Tagliaferri
+      bio: Lisa is a developer education engineer at Sourcegraph.
```

Comment:
Does Lisa want her title etc here?

Reply:
Would we be opposed to is just being `Director of Developer Education, Sourcegraph` or something like that without repetition of the name? I'll edit it to that

Reply:
@ltagliaferri could you let us know what we should add to your bio.

Reply:
I added it in [this commit](https://github.com/sourcegraph/learn/pull/136/commits/2c282341d7b9d946e8c1776a41c2d3f553159fdd)